documentation/synthetic_tokens/security/bug_bounty.md

```diff
@@ -0,0 +1,78 @@
+# UMA Bug Bounty Program
+
+Security of the platform is our highest priority. 
+All contract code and balances are publicly verifiable, and security researchers are eligible for a bug bounty for reporting undiscovered vulnerabilities.
+
+## Audit
+<!-- TODO: Add details of audit. 
+The contracts have been carefully audited by smart contract security experts [names]. 
+The audit report can be found [here].
+--> 
+
+UMA encourages the community to audit our contracts and security; we also encourage the responsible disclosure of any issues.
+The bug bounty program is intended to recognize the value of working with the community of independent security researchers, and sets out our definition of good faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
+
+## Rewards
+UMA offers substantial rewards for discoveries that can prevent the loss of assets, the freezing of assets, or harm to users. 
+
+Rewards range from $500 to $25,000. 
+All rewards will be paid in ETH, and the amount of compensation will vary depending on severity. 
+The severity is calculated according to the [OWASP](https://owasp.org/www-project-risk-assessment-framework/) risk rating model based on Impact and Likelihood.
+
+![](severity.png)
+
+## Scope
+<!-- TODO: add version name of DVM 
+The scope of our bug bounty program includes core smart contracts related to release [v1.0] of the DVM and financial templates. 
+-->
```

Comment:
Are we agreed on using v1.0 as the version name for the DVM? @mrice32 @hlambur @allisonlu22

Reply:
SGTM

Reply:
Great, will uncomment this. Aside from this, I think this is ready to be merged, but will defer to @hlambur to merge.